### DISCUSSION TITLE
KODI Matrix Addon for switching HyperHDR SDR/HDR tone mapping?

### DISCUSSION THREAD

**Discussion by menotuu** [awawa-dev/HyperHDR#85] [Ideas]
hello, can someone create a kodi matrix addon for switching the hdr/sdr tone mapping for HyperHDR?
would be very nice!

**Comment by menotuu** [comment]
or is there another solution?

**Comment by awawa-dev** [comment]
If Kodi doesn't provide such information about current video stream HDR status in its API then there is no chance...maybe https://kodi.wiki/view/JSON-RPC_API/v12#Player.Video.Stream could work but someone must test it first with SDR/HDR content and provide us results as I use don't Kodi. Then I could back to the implementation subject and even install Kodi ;)
Other solutions: 
- denon AVR which commands home automation system to remotely control HyperHDR tone mapping
- custom commands for home automation to remotely control HyperHDR
- using a browser's link to turn on/off HDR tone mapping from smartphone
...and one more possible solution for Raspberry Pi that will come with next version soon

**Comment by menotuu** [comment]
Oh, I didn't mean an automatism, but quite simply an app with which you can manually switch between HDR and SDR by sending the commands:
Turning HDR tone mapping OFF:
http: // IP_OF_HYPERHDR: 8090 / json-rpc? request = {% 22command% 22% 3A% 22videomodehdr% 22% 2C% 22HDR% 22% 3A0}


Turning HDR tone mapping ON:
http: // IP_OF_HYPERHDR: 8090 / json-rpc? request = {% 22command% 22% 3A% 22videomodehdr% 22% 2C% 22HDR% 22% 3A1}

But okay, you don't use Kodi.
I don't know how to use https://kodi.wiki/view/JSON-RPC_API/v12#Player.Video.Stream myself.
